airflow/operators/python_operator.py

```diff
@@ -330,13 +330,28 @@ def _write_string_args(self, filename):
 
     def _write_args(self, input_filename):
         # serialize args to file
+        if self.use_dill:
+            serializer = dill
+        else:
+            serializer = pickle
+        # some args from context can't be loaded in virtual env
+        invalid_args = set(['dag', 'task', 'ti'])
         if self._pass_op_args():
+            kwargs = {}
+            for key, value in self.op_kwargs.items():
+                try:
+                    serializer.loads(serializer.dumps(value))
+                    if key not in invalid_args:
+                        kwargs[key] = value
+                except Exception as e:
+                    msg = """
+                    "Exception %s found while serializing argument
+                    object: %s on op_kwargs key %s ...skipping..."
+                    """ % (e, value, key)
+                    self.log.debug(msg)
```

Comment:
I would raise this level. If you try to pass something, and it being dropped silently, that would be very confusing. What do you think? Any idea how often this happens?